src/rectangle_selection.jl

```diff
@@ -74,105 +79,100 @@ end
 # Mouse actions for rectangular selection creation and modification
 function init_rect_select(ctx::ImageContext)
     c = ctx.canvas
-    #rect = ctx.shape
-    # Define limits of rectangle
-    p1 = Signal(XY{UserUnit}(-1.0,-1.0))
-    p2 = Signal(XY{UserUnit}(-1.0,-1.0))
     
-    rect = map(p1,p2) do point1,point2
-        Rectangle(point1,point2)
+    # Build rectangle & points array
+    pts = Signal((XY{UserUnit}(-1.0,-1.0),XY{UserUnit}(-1.0,-1.0)))
+    rect = map(pts) do p
```

Comment:
`rect = map(p->Rectangle(p...), pts)` can also be used for short functions like this. (Optional.)